appveyor.yml

```diff
@@ -0,0 +1,26 @@
+os: Visual Studio 2015
+
+environment:
+  matrix:
+    - CLI_VERSION: 1.0.0-preview3-003857
+
+build_script:  
+
+  # .NET Core SDK binaries
+  - ps: $url = "https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.zip"
```

Comment:
because i used that lot a long time, so i try to avoid pain to others. but as all suggestion, is just a suggestion.
It's the same reason why we use paket to pin dependencies version with lock file, reproducible build environment